spec/draft-barnes-cfrg-4q.md

```diff
@@ -662,26 +665,22 @@ DH(m, P):
 Return [m]*Q in affine coordinates
 ~~~~
 
+The role of the separate multiplication by 392 is to ensure that Q is an N-torsion
+point so that the scalar multiplication algorithms above may be used safely to produce
+correct results.  In other words, as the cofactor is greater than one, Curve4Q MUST NOT be used with
+ordinary Diffie-Hellman; it MUST always be used for Diffie-Hellman
+with cofactor.  Note that the multiplication by the cofactor 392 can be safely
+computed with nine doublings and two additions via the double and add
+method. 
+
 Two users, Alice and Bob, can carry out the following steps to derive
-a shared key: both pick a random string of 32 bytes, mA and mB
+a shared key: both pick any random string of 32 bytes, mA and mB,
 respectively. Alice computes the public key A = Compress(DH(mA, G)),
 and Bob computes the public key B = Compress(DH(mB, G)). They exchange
 A and B, and then Alice computes KAB = DH(mA, Expand(B)) while Bob
 computes KBA = DH(mB, Expand(A)), which produces the shared point K =
-KAB = KBA.
-
-The y coordinate of K, represented as a 32 byte string as in the
-section "Curve Points" is the shared secret. The x coordinate computed
-doesn't matter for the value of this shared secret. The length of the
-shared secret is always 32 bytes.
-
-Note that the multiplication by the cofactor 392 can be safely
-computed with nine doublings and two additions via the double and add
-method. It MUST NOT be computed with either algorithm above,
-as P is not known to be a N-torsion point, and therefore the scalar
-recoding or addition of N will produce the wrong answer. The role of
-the separate multiplication by 392 is to ensure that Q is an N-torsion
-point so that the algorithms above may be used.
+KAB = KBA. The y coordinate of K, represented as a 32 byte string as detailed in the
```

Comment:
Stupid question: Why are we using the `y` coordinate?  I thought it was more traditional to use `x`.  I realize that in the Edwards context they're basically the same (unlike the cube/square difference in Weierstraß form).  I'm just concerned that this could cause cognitive friction for a lay reader.